tests/testthat/test-layers.R

```diff
@@ -1,21 +1,169 @@
 context("layers")
 
+assets <- normalizePath(testthat::test_path("test-assets"), mustWork = TRUE)
+
 blue <- list(color = "blue !default")
 red <- list(color = "red !default")
 green <- list(color = "green !default")
-core <- sass_layer(before = blue, after = "body { color: $color; }")
+core <- sass_layer(
+  defaults = blue,
+  declarations = "@function invert($color, $amount: 100%) {
+    $inverse: change-color($color, $hue: hue($color) + 180);
+    @return mix($inverse, $color, $amount);
+  }",
```

Comment:
DYK sass has a built in `invert()` that appears to do the same? 

```
> sass::sass("@debug invert(blue)")
stdin:1 DEBUG: yellow
```

Reply:
Note to self: use a different name